nextflow.config

```diff
@@ -35,41 +35,66 @@ params {
 
 profiles {
 
-  standard {
-        params.cloudProcess = false
-        params.cloudDatabase = false
-        includeConfig 'configs/local.config'
-  }
-
- lsf {
-        params.cloudProcess = true
-        includeConfig 'configs/lsf.config'
- }
-
- ebi {
-        params.cloudProcess = true
-  	  params.workdir = "/hps/nobackup2/production/metagenomics/$USER/nextflow-work-$USER"
-  	  params.cloudDatabase = "/homes/$USER/data/nextflow-databases/"
-  	  params.cachedir = "/hps/nobackup2/singularity/$USER"	
-        includeConfig 'configs/lsf.config'
- }
-
- git_action {
-        params.cloudProcess = false
-        params.cloudDatabase = false
-        includeConfig 'configs/local.config'
-        params.dv = true
-        params.ma = true
-        params.mp = true
-        params.vf = false
-        params.vs = true
-        params.pp = true
-        params.sm = true
-        params.vb = true
-        params.vn = true
-        params.anno = true
-        params.cores = 2
-        params.fasta = 'test-data/all_pos_phage.fa'
- }
+       local {
+              workDir = params.workdir
+              params.cloudProcess = false
+              includeConfig 'configs/local.config'
+       }
+
+       lsf {
+              executor {
+                     name = "lsf"
+                     queueSize = 200
+              }
+              workDir = params.workdir
+              params.cloudProcess = true
+              process.cache = "lenient"
+              includeConfig 'configs/node.config'
+       }
+
+       docker {
+              docker { enabled = true }
+              includeConfig 'configs/container.config'
+       }
+
+       singularity {
+              singularity { 
+                     enabled = true
+                     autoMounts = true
+                     cacheDir = params.cachedir
+              }
+              includeConfig 'configs/container.config'
+       }
+
+       git_action {
+              params.cloudProcess = false
+              includeConfig 'configs/local.config'
+              params {
+                     ma = true
+                     mp = true
+                     vf = false
```

Comment:
out of curiosity: why is VirFinder false for the git action?

Reply:
smallest run without a database, so we actually run a virfinder during gitaction till samtools so part of WtP is running with real data